frontend/src/components/QuestionInput/FileUploadPreviewContainer.test.tsx

```diff
@@ -5,6 +5,13 @@ import "@testing-library/jest-dom";
 
 import { FilePreview } from "../../utils/fileUploadUtils";
 
+jest.mock("framer-motion", () => ({
+  motion: {
+    div: (props: any) => <div data-testid="motion-div" {...props} />,
```

Comment:
pebble: I don't think you actually want to spread the props over the div element, since the majority of the props don't apply to HTML attributes, and since you're mocking it out you probably only care about `children`.

I actually wonder if this mock is actually working? because if this mock component were actually being called, then I think `props.children` would be getting passed to the div element like `<div children={children} />` which is invalid. 
```suggestion
    div: (props: { children: React.ReactNode }) => <div data-testid="motion-div">{children}</div>,
```
But maybe the JSX parser is being forgiving and is taking the children attribute on the HTML element and rendering it like a normal component? Either way, probably a good idea to remove the ambiguity.